examples/vite/src/App.tsx

```diff
@@ -88,6 +90,7 @@ const App = () => {
             <Window>
               <ChannelHeader Avatar={ChannelAvatar} />
               <MessageList returnAllReadData />
+              <AITypingIndicatorView />
```

Comment:
Wouldn't the indicator cover part of the message being generated in such markdown? Would it be possible that there would be 2 typing indicators at the same time? (one for human and the other for AI)

Reply:
It would technically cover part of it, yes - the general idea is that the integrators can put it wherever they'd like. We don't want it to be part of the SDK itself - perhaps some styling fixes need to be done here ?